.travis.yml

```diff
@@ -12,6 +12,8 @@ addons:
       - debian-sid
     packages:
       - shellcheck
+      - pwgen
+      - whois
```

Comment:
Added an inline comment - the type I dislike the least, because it will be deleted with the line it affects.